style-dictionary/visual-refresh/colors.ts

```diff
@@ -62,6 +61,7 @@ const tokens: StyleDictionary.ColorsDictionary = {
   colorBackgroundNotificationGreen: '{colorGreen600}',
   colorBackgroundNotificationGrey: { light: '{colorGrey600}', dark: '{colorGrey550}' },
   colorBackgroundNotificationRed: '{colorRed600}',
+  colorBackgroundNotificationStackBar: '{colorGrey750}',
```

Comment:
What's this one for?

Reply:
There was a misuse of the app layout tokens in the stackable flashbar code which was blocking me from using a single set of tokens for both app layouts. You can check out the changes to the flashbar collapsible css.